app/controllers/foreman_inventory_upload/missing_hosts_controller.rb

```diff
@@ -0,0 +1,24 @@
+module ForemanInventoryUpload
+  class MissingHostsController < ::ApplicationController
+    def index
+      organizations = User.current.my_organizations
```

Comment:
```suggestion
      organizations = Organization.current || User.current.my_organizations
```
If the user has scoped the UI to a specific org, I think we should respect his choice here too.